internal/services/legacy/virtual_machine_unmanaged_disks_resource_test.go

```diff
@@ -322,7 +322,7 @@ func TestAccVirtualMachine_changeOSDiskVhdUri(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_virtual_machine", "test")
 	r := VirtualMachineResource{}
 
-	data.ResourceTest(t, r, []acceptance.TestStep{
+	data.ResourceTestIgnoreRecreate(t, r, []acceptance.TestStep{
```

Comment:
Again here. This test doesn't provide any value to the resource functionality as it just triggers a `ForceNew` on the schema, which will always have the same outcome. This test should just be removed.

Reply:
Removed